docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/EventSubscriber/ResponseCacheSubscriber.php

```diff
@@ -63,6 +64,18 @@ public function setHeaderCacheControl(ResponseEvent $event) {
     $response->setClientTtl($ttl);
   }
 
+  /**
+   * Clear rendered cache on save of url redirect.
+   */
+  public function redirectCacheResponse(ResponseEvent $event) {
+    $response = $event->getResponse();
+    // Check if the response is a redirect.
+    $route = $event->getRequest()->attributes->get('_route');
+    if ($response instanceof RedirectResponse && ($route === 'entity.redirect.canonical' || $route === 'redirect.add')) {
+      drupal_flush_all_caches();
```

Comment:
This should not be used in our code. I saw this for another item. This has a huge performance impact when this happen.